### PR TITLE
Includes GPU autoscaling as part of README charter

### PR DESCRIPTION
**Pull request by ellis-bigelow**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/80)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # KFServing
-KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. KFServing brings cutting edge serving features like Scale to Zero and Canary Rollouts to your ML deployments.
+KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. KFServing brings cutting edge serving features like GPU Autoscaling, Scale to Zero, and Canary Rollouts to your ML deployments.
```
Comment (rakelkar) [inline README.md]: GPU autoscaling, because knative?

Comment (ellis-bigelow) [reply README.md]: Yeah. It's quite difficult to effectively autoscale GPUs on GPU metrics. Different workloads max out GPUs at different metric values, so it's easier to just use the queue which autoscales generically.

File: docs/samples/tensorflow/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+
+# Predict on a KFService using Tensorflow
+## Setup
+1. Your ~/.kube/config should point to a cluster with KFServing installed.
+2. Your cluster's Istio Ingressgateway must be network accessible.
```
Comment (animeshsingh) [inline docs/samples/tensorflow/README.md]: Should be two words Ingress gateway

File: docs/samples/tensorflow/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+
+# Predict on a KFService using Tensorflow
+## Setup
+1. Your ~/.kube/config should point to a cluster with KFServing installed.
```
Comment (animeshsingh) [inline docs/samples/tensorflow/README.md]: Maybe hyperlink to KFServing install doc

**Comment by ellis-bigelow** [comment]
This is the istio fix I was talking about. Istio is preventing outbound
traffic to the gcs auth metadata service. It's actually a bug in istio to
configure directly, so the only way is to open up outbound traffic at
either the cluster or pod level.

*From: *Dan Sun <notifications@github.com>
*Date: *Sun, May 12, 2019, 12:08 PM
*To: *kubeflow/kfserving
*Cc: *Ellis Bigelow, Author

*@yuzisun* commented on this pull request.

**Comment by ellis-bigelow** [comment]
@yuzisun @animeshsingh any other comments?

File: docs/samples/tensorflow/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+```
+MODEL_NAME=flowers-sample
+INPUT_PATH=@./input.json
+CLUSTER_IP=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+
+curl -v -H "Host: flowers-sample.default.svc.cluster.local" http://$CLUSTER_IP/v1/models/$MODEL_NAME:predict -d $INPUT_PATH
+```
+Expected Output
+```
+*   Trying 34.83.190.188...
```
Comment (animeshsingh) [inline docs/samples/tensorflow/README.md]: You may want to get rid of ip addresses here

File: docs/samples/tensorflow/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+< 
+{
+    "predictions": [
+        {
+            "scores": [0.999115, 9.20988e-05, 0.000136786, 0.000337257, 0.000300533, 1.84814e-05],
+            "prediction": 0,
+            "key": "   1"
+        }
+    ]
+* Connection #0 to host 34.83.190.188 left intact
```
Comment (animeshsingh) [inline docs/samples/tensorflow/README.md]: Same as above

File: docs/samples/tensorflow/README.md (hunk 4 above)
Comment (ellis-bigelow) [reply docs/samples/tensorflow/README.md]: Eh I'm not concerned. These are publically assignable GKE cluster IPs. I don't even own this anymore.

**Pull request merged**